scripts/opm_status.coffee

```diff
@@ -24,5 +24,5 @@ module.exports = (robot) ->
         msg.send "Well, what does Capital Weather Gang say?"
       else
         status = JSON.parse(body)
-        msg.send status['Icon'] + ' ' + icons[status['Icon']] + ' - ' + status['StatusSummary'] + ' - ' + '<' + status['Url']+ '|More>'
+        msg.send status['Icon'] + ' ' + icons[status['Icon']] + ' - ' + status['StatusSummary'] + ' Read More: ' + status['Url']
```

Comment:
Just FYI (in case you weren't aware) that CoffeeScript has string interpolation:

http://coffeescript.org/#strings

If you prefer it this way, feel free to merge!